src/main/resources/templates/template_hmpps-approved-premises-api.mustache

```diff
@@ -744,10 +775,281 @@
+            <h4>Domain event</h4>
+            <table class="summary-list">
+                <tr><td>Created at</td><td>{{ optionalValue (formatDate created_at) }}</td></tr>
+                <tr><td>Trigger source</td><td>{{ optionalValue trigger_source }}</td></tr>
+                <tr><td>Triggered by user</td><td>{{ getUserLastName triggered_by_user }}</td></tr>
+                <tr><td>Occurred at</td><td>{{ optionalValue (formatDate occurred_at) }}</td></tr>
+                <tr><td>Service</td><td>{{ optionalValue service }}</td></tr>
+                <tr><td>Updated at</td><td>{{ optionalValue (formatDate data.eventDetails.updatedAt) }}</td></tr>
+            </table>
+            <h5>Updated by</h5>
+            <table class="data-table">
+                <tr>
+                    <td class="data-column-25">Origin</td>
+                    <td class="data-column-25">Name</td>
+                    <td class="data-column-25">Email</td>
+                    <td class="data-column-25">Username</td>
+                </tr>
+                <tr>
+                    <td>{{ optionalValue data.eventDetails.updatedBy.origin }}</td>
+                    <td>{{ optionalValue data.eventDetails.updatedBy.name }}</td>
+                    <td>{{ optionalValue data.eventDetails.updatedBy.email }}</td>
+                    <td>{{ optionalValue data.eventDetails.updatedBy.username }}</td>
+                </tr>
+            </table>
+            <h5>New status</h5>
+            <table class="summary-list">
+                <tr><td>Name</td><td>{{ optionalValue data.eventDetails.newStatus.name }}</td></tr>
+                <tr><td>Description</td><td>{{ optionalValue data.eventDetails.newStatus.description }}</td></tr>
+                <tr><td>Label</td><td>{{ optionalValue data.eventDetails.newStatus.label }}</td></tr>
+            </table>
+            <p><b>Status details</b></p>
+            {{#data.eventDetails.newStatus.statusDetails}}
+            <table class="data-table">
+                <tr>
+                    <td class="data-column-50">Name</td>
+                    <td class="data-column-50">Label</td>
+                </tr>
+                <tr>
+                    <td>{{ optionalValue name }}</td>
+                    <td>{{ optionalValue label }}</td>
+                </tr>
+            </table>
+            {{/data.eventDetails.newStatus.statusDetails}}
+            {{^data.eventDetails.newStatus.statusDetails}}
+                <p>No Data Held</p>
+            {{/data.eventDetails.newStatus.statusDetails}}
+        {{/DomainEvents}}
+        {{^DomainEvents}}
+            <p>No Data Held</p>
+        {{/DomainEvents}}
+
     {{/ShortTermAccommodation}}
-    {{^ShortTermAccommodation}}
+
+    {{#ApprovedPremises}}
+    <h2>Approved Premises</h2>
+
+        <h3>Booking Extensions</h3>
+        {{#BookingExtensions}}
+            <table class="summary-list">{{#each .}}<tr><td>{{ @key }}</td><td>{{ this }}</td></tr>{{/each}}</table><br/>
```

Comment:
will just output the data in a table using key value pairs for now as we don't know what the format is, tried to use partials to reduce repetition in sections below but couldn't quite get that to work